dockerfiles/alpine/is/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk11:jdk-11.0.8_10-alpine
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.9_11
```

Comment:
In the case of this architecture addition x86_64, shall we maintain the same pattern used for the other simultaneous releases?

Reply:
+1